src/MSALWrapper/AuthFlow/AuthFlowFactory.cs

```diff
@@ -34,9 +35,18 @@ public class AuthFlowFactory
             string preferredDomain,
             string promptHint,
             string osxKeyChainSuffix,
-            IPCAWrapper pcaWrapper = null)
+            IPCAWrapper pcaWrapper = null,
+            IPlatformUtils platformUtils = null)
         {
+            logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            platformUtils = platformUtils ?? new PlatformUtils(logger);
             List<IAuthFlow> flows = new List<IAuthFlow>();
+
+            if (authMode.IsBroker() && platformUtils.IsWindows10Or11())

```

Comment:
Unless I'm mistaken I think this will just silently do nothing on a pre-Win10 platform, which might be confusing.

Reply:
Discussed offline, and we are going to not solve this now. But consider coming back to make `AuthMode` more Platform aware. See #55 .